packages/webcomponents/src/components/checkout/new-payment-method.tsx

```diff
@@ -108,7 +108,9 @@ export class NewPaymentMethod {
       <div class="payment-method">
         <div
           class={`payment-method-header p-3`}
-          onClick={() => this.onPaymentMethodOptionClick()}>
+          onClick={() => this.onPaymentMethodOptionClick()}
+          part="payment-method-header"
```

Comment:
I think we should give this a more generic name in case we do this ui pattern somewhere else. Such as `radio-list` and `radio-list-header`, or maybe there is a more standard bootstrap way of accomplishing this